app/models/like.rb

```diff
@@ -0,0 +1,10 @@
+class Like < ApplicationRecord
+  # Associations
+  belongs_to :author, class_name: 'User'
+  belongs_to :post, class_name: 'Post'
+
+  # Methods
+  def update_post_likes_counter
```

Comment:
[OPTIONAL] It is a good practice to use rails callbacks like `after_save` in scenarios like this to call the custom method after creation.

Reply:
Addressed